nomad/grapl-core.nomad

```diff
@@ -1026,22 +1065,25 @@ job "grapl-core" {
       driver = "docker"
 
       config {
-        image = "${var.container_registry}grapl/sysmon-generator:${var.sysmon_generator_tag}"
+        image = "${var.container_registry}grapl/${var.container_repo}sysmon-generator:${var.sysmon_generator_tag}"
+      }
+
+      template {
+        data        = local.local_vars
+        destination = "sysmon.env"
+        env         = true
       }
 
       env {
-        DEST_BUCKET_NAME            = var.unid_subgraphs_generated_bucket
-        DEAD_LETTER_QUEUE_URL       = var.sysmon_generator_dead_letter_queue
-        SOURCE_QUEUE_URL            = var.sysmon_generator_queue
-        GRAPL_AWS_ENDPOINT          = local.aws_endpoint
-        GRAPL_AWS_ACCESS_KEY_ID     = var.aws_access_key_id
-        GRAPL_AWS_ACCESS_KEY_SECRET = var.aws_access_key_secret
-        AWS_DEFAULT_REGION          = var.aws_region # boto3 prefers this one
-        AWS_REGION                  = var.aws_region
-        REDIS_ENDPOINT              = local.redis_endpoint
-        GRAPL_LOG_LEVEL             = var.rust_log
-        RUST_LOG                    = var.rust_log
-        RUST_BACKTRACE              = local.rust_backtrace
+        DEST_BUCKET_NAME      = var.unid_subgraphs_generated_bucket
+        DEAD_LETTER_QUEUE_URL = var.sysmon_generator_dead_letter_queue
+        SOURCE_QUEUE_URL      = var.sysmon_generator_queue
+        AWS_DEFAULT_REGION    = var.aws_region # boto3 prefers this one
+        AWS_REGION            = var.aws_region
```

Comment:
Sysmon is Rust, so keep `AWS_DEFAULT_REGION`, ditch the `boto3` comment, and ditch `AWS_REGION`.

Reply:
So actually rust throws errors if you use AWS_DEFAULT_REGION instead of AWS_REGION, so I've switched rust to use AWS_REGION